src/components/Header/Header.jsx

```diff
@@ -22,32 +23,22 @@ export class _Header extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = { openMenu: false };
+    this.state = {
+      openMenu: false,
+      hideLoginLink: false,
+      hideLanguageSwitch: false,
+    };
   }
 
   isLoggedIn() {
-    return this.variation() === LOGGED_IN_VAL;
+    return this.variation() === VARIATION_NAMES.LOGGED_IN;
   }
 
   /**
    * Content rendered within <Menu>, after the list of links
    * @returns {Node}
    */
-  afterMenuLinks() {
-    if (this.isLoggedIn()) {
-      const { initialLanguage = 'en', primaryDomain } = this.props;
-      const i18nOptions = { lng: initialLanguage };
-
-      return (
-        <a
-          className="hc-c-menu__link ds-u-border-top--1 ds-u-margin-x--1 ds-u-padding-x--0"
-          href={`${primaryDomain || ''}/logout`}
-        >
-          {translateWrapper('header.logout', i18nOptions)}
-        </a>
-      );
-    }
-  }
+  afterMenuLinks() {}
```

Comment:
[suggestion] Consider removing this since it is an empty function. With the way you've refactored, you could also remove the `afterLinks` prop from the `<Menu>` element on line 137 as well